index.html

```diff
@@ -254,27 +254,23 @@ <h2>
         The API defined in this specification is used to retrieve the
         geographic location of a hosting device. In almost all cases, this
         information also discloses the location of the user of the device,
-        thereby potentially compromising the user's privacy. A conforming
-        implementation of this specification MUST provide a mechanism that
-        protects the user's privacy and this mechanism SHOULD ensure that no
-        location information is made available through this API without the
-        user's express permission.
+        thereby potentially compromising the user's privacy. User agents MUST
+        NOT send location information to Web sites without the express
+        permission of the user.
       </p>
       <section id="privacy_for_uas">
         <h3>
-          Privacy considerations for implementers of the Geolocation API
+          Privacy considerations for geolocation permission UI
         </h3>
         <p>
-          User agents MUST NOT send location information to Web sites without
-          the express permission of the user. User agents MUST acquire
-          permission through a user interface, unless they have prearranged
-          trust relationships with users, as described below. The user
-          interface MUST include the {{URL/host}} component of the document's
-          URL. Those permissions that are acquired through the user interface
-          and that are preserved beyond the current browsing session (i.e.
-          beyond the time when the <a>browsing context</a> is navigated to
-          another URL) MUST be revocable and user agents MUST respect revoked
-          permissions.
+          User agents MUST acquire permission through a user interface, unless
+          they have prearranged trust relationships with users, as described
+          below. The user interface MUST include the {{URL/host}} component of
+          the document's URL. Those permissions that are acquired through the
+          user interface and that are preserved beyond the current browsing
+          session (i.e. beyond the time when the <a>browsing context</a> is
+          navigated to another URL) MUST be revocable and user agents MUST
```

Comment:
The idea that users MUST be able to revoke permissions they've granted belongs in the generic Permissions spec, and I think we're missing some wording about it from there. However, we have to be a bit more subtle than this to accommodate device policy, where users might not be able to revoke all permissions. This might be part of the question to answer in https://github.com/w3c/permissions/issues/231.

Reply:
Agree.